Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -146,15 +157,17 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateColor();
 				UpdatePlaceholder();
 			}
-			else if (e.PropertyName == PlatformConfiguration.iOSSpecific.Entry.AdjustsFontSizeToFitWidthProperty.PropertyName)
+			else if (e.PropertyName == Specifics.AdjustsFontSizeToFitWidthProperty.PropertyName)
 				UpdateAdjustsFontSizeToFitWidth();
 			else if (e.PropertyName == VisualElement.FlowDirectionProperty.PropertyName)
 				UpdateAlignment();
 			else if (e.PropertyName == Xamarin.Forms.InputView.MaxLengthProperty.PropertyName)
 				UpdateMaxLength();
 			else if (e.PropertyName == Entry.ReturnTypeProperty.PropertyName)
 				UpdateReturnType();
-			else if (e.PropertyName == Entry.CursorPositionProperty.PropertyName || e.PropertyName == Entry.SelectionLengthProperty.PropertyName)
+			else if (e.PropertyName == Entry.CursorPositionProperty.PropertyName)
```

Comment:
Why make this change?

Reply:
It's not really necessary anymore. I had split them apart in an earlier iteration so I could set the respective `...Pending` bools here, then later refactored that out.